gen/cheader.tmpl

```diff
@@ -294,6 +297,16 @@ typedef struct WGPU{{.Name | PascalCase}}CallbackInfo{{$.ExtSuffix}} {
     WGPU_NULLABLE void* userdata1;
     WGPU_NULLABLE void* userdata2;
 } WGPU{{.Name | PascalCase}}CallbackInfo{{$.ExtSuffix}} WGPU_STRUCTURE_ATTRIBUTE;
+
+#define WGPU_{{.Name | ConstantCase}}_CALLBACK_INFO{{$.ExtSuffix}}_INIT _wgpu_MAKE_INIT_STRUCT(WGPU{{.Name | PascalCase}}CallbackInfo{{$.ExtSuffix}}, { \
+    /*.nextInChain=*/NULL _wgpu_COMMA \
+{{-   if eq .Style "callback_mode" }}
+    /*.mode=*/WGPUCallbackMode_AllowProcessEvents _wgpu_COMMA \
```

Comment:
Default should be `WaitAnyOnly` (that's the most "basic" - allows the fewest behaviors)...
or it should be `Undefined` so the caller has to set it explicitly.

Reply:
Dawn currently defaults to `WaitAnyOnly`.

Reply:
I agree with `WaitAnyOnly`, at least for didactic purpose it is good to have a non-undefined mode by default.